### PR TITLE
docs: update READMEs

### PR DESCRIPTION
**Pull request by erickzhao**
Updates the old README we had from the beta.electronjs.org stage, and ports over the old [CONTRIBUTING.md from electron/i18n](https://github.com/electron/i18n/blob/master/contributing.md).

### CODE REVIEW

**Comment by erickzhao** [comment]
- [ ] After this PR lands, we should update the description in https://crowdin.com/project/electron to link to this new i18n.md

File: i18n.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### Translating
+
+Here are some guidelines to keep in mind as a translator:
+
+- Don't open pull requests to translate Electron; instead, do all translations on Crowdin.
+- Do not translate JavaScript keywords like `String`, `Event`, `Array`, `Class`, etc.
+- Do not translate Electron classes, method names, event names, etc.
+- If you find an error in the source English docs, open a pull request on the
+  [electron/electron][] repository, not on the website repository.
```
Comment (dsanders11) [inline i18n.md:34]: Note for future us, I bet we could make a GitHub actions workflow that automatically closes PRs to `docs/` on this repo with a note to do them upstream.

File: i18n.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+To get started, visit **[crowdin.com/project/electron][Crowdin]** and log in with your GitHub account.
+
+### Proofreading
+
+If you would like to be promoted to proofreader status on the Electron project, please translate
+1000 words or more before applying. When contacting us, please include a link to your existing
```
Comment (dsanders11) [inline i18n.md:41]: nit: contact us how?

File: i18n.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Electron Internationalization
+
+🌎 🌍 🌏
```
Comment (dsanders11) [inline i18n.md:3]: This is the most random comment to make, but I think if you technically wanted the order to reflect the Earth rotating, it should be the other way around. 😛

🌏 🌍 🌎

File: README.md [hunk 4]
```diff
@@ -1,89 +1,59 @@
...
-If you want to use the contents from [`electron/electron`][] run the following:
-
-```console
-yarn pre-build
-yarn start
-```
+## Local development
 
-If you want the website to pick your local documentation, run:
+If you want modify Electron's documentation locally and preview how it looks on the website, you can
```
Comment (dsanders11) [inline README.md:44]:
```suggestion
If you want to modify Electron's documentation locally and preview how it looks on the website, you can
```

File: README.md [hunk 5]
```diff
@@ -1,89 +1,59 @@
...
-
-The translation work happens in [Crowdin] and the changes are picked up periodically every
-15 minutes.
+### English docs
 
-Any documentation changes done manually in this repo will be eventually override by any of these
-updates.
+For English documentation, the documentation is mirrored from the [electron/electron][] repo.
+Changes are picked automatically by the [electron-website-updater][] webhook as soon as they
+are made the latest stable release branch.
```
Comment (dsanders11) [inline README.md:27]: nit: I think we should use backticks around repo names, and be consistent with naming - if we refer to one as `electron/electron`, then it should also be `electron/electron-website-updater` for the other, or vice versa (with a bare `electron`).

**Review by dsanders11 — APPROVED** [review]

**Pull request merged**